packages/samples/vuejs/README.md

```diff
@@ -1,46 +1,157 @@
+
+```js
+// vue.config.js
+{
+  ...,
+  chainWebpack: (config) => {
+    // lets us parse html templates, which will be necessary for atomic components that require the native <template> tag
+    config.module
+      .rule('html')
+      .test(/\.html$/)
+      .use('html-loader')
+      .loader('html-loader');
+
+    // tells Vue to treat any components starting with 'atomic-' as native components, and therefore not try and create Vue components out of them
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => ({
+        ...options,
+        compilerOptions: {
+          isCustomElement: (tag) => tag.startsWith('atomic-'),
+        },
+      }));
+  }
+}
+```
 
-## Type Support for `.vue` Imports in TS
+### Add dependencies
 
-TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+`html-loader` is necessary for the configuration above to work and
+`ncp` is a utility that we will use to copy some assets from the atomic project into the vue project in the next step
+```sh
+npm install --save-dev ncp html-loader
+```
 
-If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:
+### Add scripts to package.json
+In order for you to be able to leverage everything that atomic has to offer, you will need to add the following scripts to your `package.json` file:
+```json
+scripts: {
+  ...,
+  "dev": "npm run copy:assets && npm run serve",
+  "copy:assets": "ncp node_modules/@coveo/atomic/dist/atomic/assets public/assets && ncp node_modules/@coveo/atomic/dist/atomic/lang public/lang"
+}
+```
+The `dev` script assumes you are using the default `serve` script for running your application.
 
-1. Disable the built-in TypeScript Extension
-    1) Run `Extensions: Show Built-in Extensions` from VSCode's command palette
-    2) Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`
-2. Reload the VSCode window by running `Developer: Reload Window` from the command palette.
 
-## Customize configuration
+> **Note:** If you are trying to use `@coveo/atomic` in a project that runs on [vite](https://vitejs.dev/), as per [this tutorial](https://vuejs.org/guide/quick-start.html#with-build-tools), you might need to [import atomic via cdn](https://vuejs.org/guide/quick-start.html#with-build-tools), as currently there are issues setting up stencils projects using this build tool. In this case, you won't need to add the assets as they will already be available.
```

Comment:
For this note, we should add a complete example of what it would look like to use the CDN, and how the configuration would be different.

We could even have two complete different section in the REAME, one about importing from the CDN + config, and one about importing from node_modules + config.